src/intern.ts

```diff
@@ -20,39 +20,93 @@ export class intern {
     }
   }
 
-  static async findKnex(ent: any, q: any): Promise<any> {
+  static async findKnex(ent: any, q: any, knex: any): Promise<any> {
     const ent_table = intern.tablenameUtil(ent)
     const entp = intern.makeentp(ent)
 
     const isQArray = Array.isArray(q)
 
     const filter = intern.isObjectEmpty(q) ? {...entp} : isQArray ? q : {...q}
 
+    let sort = null
+    let skip = null
+    let limit = null
+
+    if (filter.sort$) {
+
+      const firstKey = Object.keys(filter.sort$)[0];
+      const sortValue = filter.sort$[firstKey] == 1 ? 'ASC' : 'DESC'
+      sort = {
+        field: firstKey,
+        order: sortValue
+      }
+
+      delete filter.sort$
+    }
+
+    if (filter.skip$) {
+      skip = filter.skip$ > 0 ? filter.skip$ : 0
+      delete filter.skip$
+    }
+
+    if (filter.limit$) {
+      limit = filter.limit$ > 0 ? filter.limit$ : null
+      delete filter.limit$
+    }
+
     const args = {
       table_name: ent_table,
       data: intern.isObjectEmpty(filter) ? false : filter,
-      isArray: isQArray
+      isArray: isQArray,
+      sort,
+      skip,
+      ...(limit && {limit})
     }
     
-    const query = await Q.select(args)
+    const query = await qBuilder(knex).select(args)
```

Comment:
intead of creating a new QBuilder each time, just create it once, after you get the knex object

Reply:
Hmmm, like this?
```
const builder = qBuilder(knex)

....

const query = await builder.select(args)
```